backend/geonature/core/gn_synthese/blueprints/observer.py

```diff
@@ -0,0 +1,175 @@
+from warnings import warn
+
+from flask import Blueprint, g, jsonify, request
+
+from geonature import app
+from geonature.utils.env import db, DB
+
+from geonature.core.gn_commons.models import TMedias
+
+from geonature.core.gn_permissions import decorators as permissions
+from geonature.core.gn_permissions.decorators import login_required
+from geonature.core.gn_synthese.models import (
+    CorAreaSynthese,
+    Synthese,
+    CorObserverSynthese,
+)
+from geonature.core.gn_commons.models import TMedias
+from geonature.core.gn_synthese.utils.observer_sheet import ObserverSheetUtils
+from geonature.core.gn_synthese.utils.query_select_sqla import SyntheseQuery
+
+from ref_geo.models import BibAreasTypes, LAreas
+from utils_flask_sqla.response import json_resp
+
+
+from sqlalchemy import distinct, func, select
+from werkzeug.exceptions import BadRequest
+from geonature.core.gn_synthese.utils.taxon_sheet import TaxonSheet, TaxonSheetUtils, SortOrder
+from apptax.taxonomie.models import Taxref
+
+observer_info_routes = Blueprint("synthese_observer_info", __name__)
+
+if app.config["SYNTHESE"]["ENABLE_OBSERVER_SHEETS"]:
+
+    @observer_info_routes.route("/observer_stats/<int:id_role>", methods=["GET"])
+    @permissions.check_cruved_scope("R", get_scope=True, module_code="SYNTHESE")
+    @json_resp
+    def observer_stats(scope, id_role):
+        """Return stats for a specific taxon"""
+
+        # Handle area type
+        area_type = request.args.get("area_type")
+
+        if not area_type:
+            raise BadRequest("Missing area_type parameter")
+
+        # Ensure area_type is valid
+        valid_area_type = (
+            db.session.query(BibAreasTypes.type_code)
+            .distinct()
+            .filter(BibAreasTypes.type_code == area_type)
+            .scalar()
+        )
+        if not valid_area_type:
+            raise BadRequest("Invalid area_type")
+
+        # Subquery to fetch areas based on area_type
+        areas_subquery = (
+            select(LAreas.id_area)
+            .where(LAreas.id_type == BibAreasTypes.id_type, BibAreasTypes.type_code == area_type)
+            .alias("areas")
+        )
+
+        # Observer subquery
+        observer_subquery = ObserverSheetUtils.get_observers_subquery(id_role)
+
+        # Main query to fetch stats
+        query = (
+            select(
+                [
+                    func.count(distinct(Synthese.id_synthese)).label("observation_count"),
+                    func.count(distinct(Synthese.cd_nom)).label("taxa_count"),
+                    func.count(distinct(areas_subquery.c.id_area)).label("area_count"),
+                    func.min(Synthese.date_min).label("date_min"),
+                    func.max(Synthese.date_max).label("date_max"),
+                ]
```

Comment:
```suggestion
                    func.count(distinct(Synthese.id_synthese)).label("observation_count"),
                    func.count(distinct(Synthese.cd_nom)).label("taxa_count"),
                    func.count(distinct(areas_subquery.c.id_area)).label("area_count"),
                    func.min(Synthese.date_min).label("date_min"),
                    func.max(Synthese.date_max).label("date_max"),
```

pas besoin de brackets dans la syntaxe sqla 2.x